web/src/components/Inputs/RichText.js

```diff
@@ -40,6 +40,7 @@ class RichText extends Component {
         editorState={this.state.content}
         onEditorStateChange={this.onEditorChange}
         onBlur={this.onBlur}
+        editorClassName={this.props.editorClassName || 'rte-textarea-m'}
```

Comment:
`editorClassName` is something that [React Draft WYSIWYG expects](https://jpuri.github.io/react-draft-wysiwyg/#/docs?_k=jjqinp), but we weren't previously using it.